.github/workflows/image-scan-analysis.yml

```diff
@@ -1,10 +1,8 @@
 name: Image Security Scan
 
 on:
-  push:
-    branches: [master, test, dev]
-  pull_request:
-    branches: [master, test, dev]
+  pull_request_target:
+    types: [assigned, opened, synchronize, reopened]
```

Comment:
What was the motivation behind this? wouldn't this run the action once if a PR was opened, and then again when it was assigned?